jupyter-nextflow/Dockerfile

```diff
@@ -1,55 +1,51 @@
-# Copyright (c) Jupyter Development Team.
-# Distributed under the terms of the Modified BSD License.
-FROM quay.io/cdis/scipy-notebook:2023-03-13
+############################ Jupyter-nextflow With AzLinux ###################
+ARG ROOT_CONTAINER=quay.io/cdis/azlinux-jupyter-scipy:azlinux-scipy-master
```

Comment:
Created a new tag called `azlinux-scipy-master` in quay [here](https://quay.io/repository/cdis/jupyter-notebook?tab=tags) which is a duplicate of the quay image `azlinux-scipy-chore_change_nf_base_img`